bokeh/transforms/image_downsample.py

```diff
@@ -11,20 +11,21 @@
 
 
 def source(**kwargs):
-  kwargs['transform'] = {'resample':'heatmap',
-                         'global_x_range' : [0, 10],
-                         'global_y_range' : [0, 10],
-                         'global_offset_x' : [0],
-                         'global_offset_y' : [0],
-                         'type' : 'ndarray'
-
-  }
-  kwargs['data'] = {'x': [0],
-                    'y': [0],
-                    'dw' : [10],
-                    'dh' : [10],
-                   }
-  return ServerDataSource(**kwargs)
+    kwargs['transform'] = {'resample':'heatmap',
+                           'global_x_range' : [0, 10],
+                           'global_y_range' : [0, 10],
+                           'global_offset_x' : [0],
+                           'global_offset_y' : [0],
+                           'type' : 'ndarray'
```

Comment:
Trailing comma recommended on last lines. FWIW I really hate the aligned brace style. I'd personally rather see
```
kwargs['transform'] = {
    'resample':'heatmap',
    'global_x_range' : [0, 10],
    'global_y_range' : [0, 10],
    'global_offset_x' : [0],
    'global_offset_y' : [0],
    'type' : 'ndarray',
}
```
or
```
kwargs['transform'] = dict(
    resample='heatmap',
    global_x_range=[0, 10],
    global_y_range=[0, 10],
    global_offset_x=[0],
    global_offset_y=[0],
    type'='ndarray',
)
```